Berksfile

```diff
@@ -1,3 +1,5 @@
 source "https://supermarket.getchef.com"
 
 metadata
+
+cookbook 'config-driven-helper', git: 'https://github.com/inviqa/chef-config-driven-helper.git', ref: '1.4.1'
```

Comment:
This shouldn't be here. If testing needs config-driven-helper, it should either use it's own Berksfile in a subfolder, or config-driven-helper uploaded to supermarket.chef.io (although I've already been considering doing that anyway)

Adding it as a Berksfile restriction will further restrict the metadata.rb version when Berkshelf used for the parent project.